cinema/views.py

```diff
@@ -1,45 +1,115 @@
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from rest_framework import status
+from rest_framework import status, generics, mixins, viewsets
```

Comment:
imports style - each on new line

Reply:
have problem with style in each in new line on github don`t pass flake8 tests